### PR TITLE
[14.0][IMP] product_attribute_value_menu: merge module product_attribute_value_menu from V13

### PR DESCRIPTION
**Pull request by JordiMForgeFlow**
As discussed and agreed with the modules authors in https://github.com/OCA/product-attribute/pull/1079, due to two modules having the same name (one is in V13 and one is in V14) we are merging their features in the same module.

### CODE REVIEW

File: product_attribute_value_menu/models/product_attribute_value.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    product_count = fields.Integer(string="Product", compute="_compute_product_count")
+
+    def _compute_product_count(self):
+        for value in self:
+            value.product_count = len(value.pav_attribute_line_ids)
+
+    def action_view_product(self):
+        action = self.env["ir.actions.act_window"].for_xml_id(
+            "product", "product_template_action"
+        )
```
Comment (GuillemCForgeFlow) [inline product_attribute_value_menu/models/product_attribute_value.py]:
```suggestion
        action = self.env["ir.actions.act_window"]._for_xml_id(
            "product.product_template_action"
        )
```
When I try to open the products the action is failing due to this.

**Review by GuillemCForgeFlow — CHANGES_REQUESTED** [review]

**Review by GuillemCForgeFlow — APPROVED** [review]
LGTM :+1:

File: product_attribute_value_menu/README.rst [hunk 2]
```diff
@@ -25,10 +25,16 @@ Product Attribute Value Price Import
 
 |badge1| |badge2| |badge3| |badge4| |badge5| 
 
-This module adds a menu item in sales > configuration where user can see/edit all extra
-prices set for each attribute value in each product template.
+This module adds:
 
-It also allows extra prices to be imported via standard csv/xlsx import.
+- A menu item in Sales > Configuration where user can see/edit all extra
```
Comment (francesco-ooops) [inline product_attribute_value_menu/README.rst]: A menu item in Sales > Configuration named *Attribute Values Extra Prices* where user can see/edit all extra

---
This is just for clarity, can you also change the menu name from "Product Variant Values" to "Attribute Values Extra Prices"?

**Review by francesco-ooops — APPROVED** [review]
@JordiMForgeFlow just a minor change, functional review ok! 👍

File: product_attribute_value_menu/README.rst (hunk 2 above)
Comment (JordiMForgeFlow) [reply product_attribute_value_menu/README.rst]: Done! :)

**Review by GuillemCForgeFlow — APPROVED** [review]
Functional review :+1:

**Comment by francesco-ooops** [comment]
@pedrobaeza could you check? fixes [this issue](https://github.com/OCA/product-attribute/pull/1079#issuecomment-1196872693)

**Pull request merged**